src/components/SiteAlert/SiteAlert.stories.tsx

```diff
@@ -25,9 +26,9 @@ const infoHeading = 'Short alert message'
 const additionalContext = (
   <p className="usa-alert__text">
     Additional context and followup information including{' '}
-    <a className="usa-link" href="#">
+    <Link className="usa-link" href="#">
```

Comment:
Swapping out `className="usa-link"` with `variant="unstyled"` will apply the `usa-link` class internally. I think we should go for that approach to encapsulate the uswds class.

Reply:
Whoops, got this logic wrong. You shouldn't need to pass a variant or a className and it should apply `usa-link` internally.